components/approved-online-courses/ApprovedOnlineCoursesClient.js

```diff
@@ -31,7 +31,7 @@ export default function ApprovedOnlineCoursesClient({ key, Courses }) {
                 onClick={() => {
                   toggleTerm(term.id);
                 }}
-                className="bg-primary-dark text-white w-1/4 font-semibold text-sm">
+                className="bg-primary-dark text-white w-1/23 font-semibold text-sm">
```

Comment:
i dont think this class even exists in tailwind.

Reply:
does, it worked for me, width now exactly like wanted, currently randomising bg images

Reply:
https://tailwindcss.com/docs/width check for yourself, there is no such class, wtv is working for you is because this width is not applying, making it a full width, or takes the width of the parent